x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/results/attack_discovery_panel/tabs/alerts_tab/ai_for_soc/wrapper.tsx

```diff
@@ -26,7 +30,8 @@ export const ERROR_TEST_ID = 'attack-discovery-alert-error';
 export const SKELETON_TEST_ID = 'attack-discovery-alert-skeleton';
 export const CONTENT_TEST_ID = 'attack-discovery-alert-content';
 
-const dataViewSpec: DataViewSpec = { title: '.alerts-security.alerts-default' };
+const AI4SOC_INDEX_PATTERN = `${DEFAULT_ALERTS_INDEX}-default`;
```

Comment:
As you had mentioned in a Slack discussion, should we append the space in the code below using the [useSpaceId](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/common/hooks/use_space_id.ts) hook instead of setting it to the `default` constant here?